DrinkApp.rb

```diff
@@ -1,43 +1,42 @@
 #!/usr/bin/env ruby
+load "./ui.rb"
 
 class DrinkApp
 drinkName = ""
 drinkDescription = ""
 wineType = ""
+#ui = UI.new()
 	def run
-		puts "Would you like wine or water?"
+		UI.new.prompt("Would you like wine or water?")
```

Comment:
`@ui.wine_or_water_message` would be an example of how you would initialize the UI once (UI.new), and instead of calling `.prompt`, stick all the prompts in methods inside of your UI class.